lib/brainstem/presenter_collection.rb

```diff
@@ -275,7 +278,16 @@ def run_search(scope, includes, sort_name, direction, options)
         search_options[:page] = calculate_page(options)
       end
 
-      search_options.reverse_merge!(extract_filters(options))
+      extracted_filters = extract_filters(options)
+      extracted_filters.each do |key, val|
+        if val[:arg].nil?
+          extracted_filters.delete(key)
```

Comment:
So search errors if it receives `key => nil`?  What do nil args represent?  Default filters?

Assuming we do need to filter them, I'd probably make a new variable called `extracted_filters_for_search` or something that is key => val[:arg] like

```ruby
extracted_filters.each.with_object({}) { |(key, val), hash| hash[key] = val[:arg] if val[:arg] }
```